bin/pd_ati_monitor.py

```diff
@@ -0,0 +1,61 @@
+import sys
+import subprocess
+import requests
+
+'''
+Usage: python pd_ati_monitor.py usr1@reg_host usr2@portal_host slack_url
+'''
+
+def sendSlack2(text, url):
+    payload = {"text":text}
+    count = 5
+    while count >0:
+        count -= 1
+        r = requests.post(url, json=payload)
+        if r.status_code == 200:
+            break
+    
+
+def getCSVUpload(usrAtHost):
+    
+    cmd = 'ssh '+usrAtHost+' grep download /home/odatsrv/run_log/upload_ati_csv_from_datastore_tables.log | wc'
```

Comment:
we should also check the modification date of these files to make sure they were updated on the current day